.pre-commit-hooks.yaml

```diff
@@ -19,3 +19,15 @@
   require_serial: true
   additional_dependencies: []
   minimum_pre_commit_version: "2.9.2"
+
+# Legacy alias
+- id: ruff
+  name: ruff (legacy alias)
```

Comment:
Retain the `id: ruff` hook ID as an alias. I've changed the name to display `ruff (legacy alias)`, but this might be too aggressive for now -- would welcome thoughts.